### DISCUSSION TITLE
Implement and test e-mail notifications

### DISCUSSION THREAD

**Discussion by vranki** [TampereHacklab/mulysa#375] [General]
Implement e-mail notifications for changes:

 - User's service subscription state changes
 - Warnings for payment expiration of subscriptions
 - (any others? add here)

Also test that the mails are really sent from production environment.

Ass settings option to DISABLE all mails to avoid spamming due to maintenance work.

**Comment by tswfi** [comment]
expiration warning emails already exist..
converting to discussion
